.nvmrc

```
@@ -0,0 +1 @@
+12.22
```

Can we just specify the major version ? This would avoid `nvm` to fail if the user only have not exactly this version.
```suggestion
12
```